docs/developer/release-process.md

```diff
@@ -97,49 +97,47 @@ helm dependency update ./chart/kubeapps
-# go get -u=patch  -t ./... # upgrades just to patch releases
+go get -u ./...
 ```
 
-> Sometimes, you may face some errors when upgrading. It is likely due to certain version requirements specified at the `replace(...)` statement in the `go.mod` file. For example, some k8s dependencies are not Go modules, so we need to manually set the version. Another reason is when using dependencies that have been renamed (eg., `docker` to `moby`) and some old transitive dependencies haven't updated that name yet.
+Then, try to manually update those versions that can be safely upgraded. A useful tool for doing so is [go-mod-upgrade](https://github.com/oligot/go-mod-upgrade).
 
-- Upgrade the [rust dependencies](../../cmd/pinniped-proxy/Cargo.toml) by running:
+#### Rust dependencies
+
+Upgrade the [rust dependencies](../../cmd/pinniped-proxy/Cargo.toml) by running:
 
 ```bash
 cd cmd/pinniped-proxy/
 cargo update
 ```
 
-- Finally, look at the [pull requests](https://github.com/kubeapps/kubeapps/pulls) and ensure there is no PR open by Snyk fixing a security issue. If so, discuss it with another Kubeapps maintainer and come to a decision on it, trying not to release with a high/medium severity issue.
+#### Security and chart sync PRs
 
-> As part of this release process, the dashboard deps _must_ be updated, the golang deps _should_ be updated, the rust deps _should_ be updated and the security check _must_ be performed.
+Finally, look at the [pull requests](https://github.com/kubeapps/kubeapps/pulls) and ensure there is no PR open by Snyk or `kubeapps-bot`fixing a security issue or bringing upstream chart changes. If so, discuss it with another Kubeapps maintainer and come to a decision on it, trying not to release with a high/medium severity issue.
 
-## 1 - Wait until the PR to the bitnami/chart repository is accepted
-
-Since the chart that we host in the Kubeapps repository is only intended for development purposes, we need to synchronize it with the official one in the [bitnami/charts repository](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps).
+> As part of this release process, the dashboard deps _must_ be updated, the golang deps _should_ be updated, the rust deps _should_ be updated and the security check _must_ be performed.
 
-To this end, our CI system will automatically (in the `sync_chart_from_bitnami` workflow, as described in the [CI documentation](./ci.md).) send a PR with the current development changes to [their repository](https://github.com/bitnami/charts/pulls) whenever a new release is triggered.
-Once the PR has been created, wait for someone from the Bitnami team to review and accept it.
+#### Send a PR with the upgrades
 
-> If the PR solely includes minor changes in the image versions, this wait can be safely skipped.
+Now create a Pull Request containing all these changes (only if only minor and patch have been bumped up) and wait until for another Kubeapps maintainer to review and accept it.
```

Comment:
```suggestion
Now create a Pull Request containing all these changes (only if only minor and patch have been bumped up) and wait until for another Kubeapps maintainer to review and accept so you can merge it.
```
Also, the "only if only minor...." is a bit confusing, I'm not sure if the second only is intentional, in which case, perhaps "only if no major versions have been bumped up", but I think you mean rather that "(there should only ever be minor and patch versions updated)"?